### ISSUE TITLE
UI of 'in your area now' section

### ISSUE THREAD

**Issue by sloorush** [nic-delhi/AarogyaSetu_Android#492]
In the **'in your area'** section, 

to view the number of COVID cases we need to wait till the box automatically scrolls. It would be a better UX if it were **manually navigated** or displayed in a **single view**.

**Comment by bastpt** [comment]
Yes. I agree. This layout of this section needs to be changed. This is the actual data people wants to see which is pretty much hiding right now as per current design. I like to contribute in this change as I am new to Kotlin language it will be tough to start right away. Is anyone willing to join in this.

**Comment by sharadcodes** [comment]
> In the **'in your area'** section,
> 
> to view the number of COVID cases we need to wait till the box automatically scrolls. It would be a better UX if it were **manually navigated** or displayed in a **single view**.

Yeah we want this, it's misleading 🤷🏻

**Comment by sharadcodes** [comment]
> Yes. I agree. This layout of this section needs to be changed. This is the actual data people wants to see which is pretty much hiding right now as per current design. I like to contribute in this change as I am new to Kotlin language it will be tough to start right away. Is anyone willing to join in this.

I can join but sadly it's all webview I guess, it's only editable by NIC team
